api/integrations/launch_darkly/services.py

```diff
@@ -316,6 +482,38 @@ def _create_features_from_ld(
     ]
 
 
+def _create_segments_from_ld(
+    ld_segments: list[tuple[ld_types.UserSegment, str]],
+    tags_by_ld_tag: dict[str, Tag],
+    project_id: int,
+) -> list[Segment]:
+    """
+
+    :param ld_segments: A list of mapping from (env, segment).
+    """
+    for ld_segment, env in ld_segments:
+        if ld_segment["deleted"]:
+            continue
+
+        Segment.objects.create(
+            name="imported-" + env + "-" + ld_segment["name"],
+            project_id=project_id,
+        )
+
+        # TODO: Tagging segments is not supported yet.
```

Comment:
cc: @khvn26 Correct me if wrong.

Reply:
Maybe we can include it in the description

Reply:
Add issue link https://github.com/Flagsmith/flagsmith/issues/3241 to the TODO and ignore for now.